src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java

```diff
@@ -137,6 +143,8 @@ public void doPost_videoHasNoTranscript_shouldReturnLectureAnyways() throws Exce
   public void doPost_urlNotInDatabase_shouldAddToDatabaseAndReturnRedirect() throws Exception {
     when(request.getParameter(LectureServlet.PARAM_LINK)).thenReturn(TEST_LINK);
     when(request.getParameter(LectureServlet.PARAM_NAME)).thenReturn(TEST_NAME);
+    when(request.getParameter(LectureServlet.PARAM_LANGUAGE))
+        .thenReturn(/* transcriptLanguage= */ "en");
```

Comment:
the string in /* =*/ should be the argument name. I assume thenReturn does not have a variable name of transcriptLanguage. For thenReturn it is obvious what the method does, so you should omit the /* =*/ comment. When I said to add this I meant for the parseAndStoreTranscript methods where it isn't obvious what the string might represent.

Reply:
gotcha, updated.